src/Fabulous.XamarinForms/Views/Controls/Image.fs

```diff
@@ -9,22 +11,35 @@ type IImage =
 module Image =
     let WidgetKey = Widgets.register<Image> ()
 
-    let Source =
-        Attributes.defineBindable<ImageSource> Image.SourceProperty
-
     let Aspect =
         Attributes.defineBindable<Aspect> Image.AspectProperty
 
+    let IsAnimationPlaying =
+        Attributes.defineBindable<bool> Image.IsAnimationPlayingProperty
+
+    let IsOpaque =
+        Attributes.defineBindable<bool> Image.IsOpaqueProperty
+
+    let Source =
+        Attributes.defineAppThemeBindable<string> Image.SourceProperty
```

Comment:
Should be of type `ImageSource`

Reply:
This needs to be a string as the setOnApptheme expects to support light and dark.

Reply:
SetOnAppTheme is not limited to string
https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.bindableobjectextensions.setonapptheme?view=xamarin-forms#xamarin-forms-bindableobjectextensions-setonapptheme-1(xamarin-forms-bindableobject-xamarin-forms-bindableproperty-0-0)

Maybe we just need to add a generic parameter to `AppThemeValue.create`?
```fs
[<Struct>]
type AppThemeValues<'T> =
    { Light: 'T
      Dark: 'T voption }
    static member inline create<'T>(light: 'T, dark: 'T) =
        { Light = light
          Dark =
              match dark with
              | None -> ValueNone
              | Some v -> ValueSome v }

AppThemeValues.create<ImageSource>(ImageSource.FromPath("light.png"), ImageSource.FromStream(darkStream))
```

Reply:
Sounds a good idea . Let me give a try

Reply:
Updated following your suggestion . Also Updated the Button to use ImageSource also